apps/meteor/app/livechat/server/lib/Contacts.ts

```diff
@@ -43,12 +55,14 @@ export const Contacts = {
 		updateUser.$set.phone = (phone && [{ phoneNumber: phone }]) || null;
 		updateUser.$set.visitorEmails = (visitorEmail && [{ address: visitorEmail }]) || null;
 
-		const allowedCF = LivechatCustomField.find({ scope: 'visitor' }, { fields: { _id: 1 } }).map(({ _id }) => _id);
+		const allowedCF = (LivechatCustomField.find({ scope: 'visitor' }, { fields: { _id: 1 } }) as ILivechatCustomField[]).map(
```

Comment:
Does this fit in one line yet? 🙈

Reply:
Yups, it seems like it does. Even if I split it into multiple lines, the prettier auto-format just reverts it back to this :bug: